### PR TITLE
Adding a header containing a share button

### LINKED ISSUE #94: Playground: Add a more visible share button
Add an icon-only button with a hover title 'Share'.

You can use this icon (from Iconator):
```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Created with Vectornator (http://vectornator.io/) -->
<svg height="100%" stroke-miterlimit="10" style="fill-rule:nonzero;clip-rule:evenodd;stroke-linecap:round;stroke-linejoin:round;" version="1.1" viewBox="0 0 20 20" width="100%" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<defs/>
<g id="Ebene-1">
<path d="M15 6L16 6C16.552 6 17 6.448 17 7L17 18C17 18.552 16.552 19 16 19L4 19C3.448 19 3 18.552 3 18L3 7C3 6.448 3.448 6 4 6L5 6M10 16L10 2" fill="none" opacity="1" stroke="#000000" stroke-linecap="butt" stroke-linejoin="miter" stroke-width="2"/>
<path d="M10 0L14 4L6 4L10 0Z" fill="#000000" fill-rule="nonzero" opacity="1" stroke="none"/>
</g>
</svg>
```

### PR DESCRIPTION
**Pull request by Lotes**
closes #94

### CODE REVIEW

File: hugo/content/playground/_index.html [hunk 1]
```diff
@@ -28,23 +28,67 @@
...
+
     const divRoot = document.getElementById("monaco-editor-root");
-    const style = "width:50%; display:inline-block; height: 100%";
-    const leftEditor = document.createElement("div");
-    leftEditor.setAttribute("style", style);
-    const rightEditor = document.createElement("div");
-    rightEditor.setAttribute("style", style);
-    divRoot.appendChild(leftEditor);
-    divRoot.appendChild(rightEditor);
+    const table = document.createElement("table");
```
Comment (spoenemann) [inline hugo/content/playground/_index.html:34]: The new layout looks good. But can't we define it in the static html file instead of creating it in JS?

Comment (Lotes) [reply hugo/content/playground/_index.html:34]: So true. Yes, I can do it.

Comment (Lotes) [reply hugo/content/playground/_index.html]: Done

**Review by spoenemann — APPROVED** [review]
Thanks!

**Pull request merged**